components/UserInput.tsx

```diff
@@ -0,0 +1,67 @@
+'use client'
+
+import { Space_Mono } from '@next/font/google'
+import { useState } from 'react';
+
+const smono = Space_Mono({
+    weight: ['400', '700'],
+    variable: '--inter-font',
+    preload: true,
+    display: 'swap',
+  })
+
+interface UserInputProps {
+    inputAmount: number;
+    value: string;
+    userInput: (value: string) => void
+}
+
+export const UserInput: React.FC<UserInputProps> = ({ inputAmount, value, userInput }) => {
+    const [textAreaValue, setTextAreaValue] = useState(value);
+    
+    return (
+        <form style={{position:'relative'}} >
+              <br/>
+              <textarea 
+                onChange={(e) => {setTextAreaValue(e.target.value)
+                    userInput(e.target.value);
+                }}
+                value={textAreaValue}  
+                id="text" name="text"
+                rows={4} cols={40}
+                maxLength={inputAmount}
+                spellCheck="false"
+                style={{
+                    width: '900px',
+                    height: '120px',
+                    fontSize: '20px',
+                    color:'white',
+                    letterSpacing: '10px',
+                    background: 'transparent',
+                    fontFamily: `${smono.style.fontFamily}`,
+                    position: 'absolute',
+                    top: '0',
+                    left: '0',
+                    zIndex: '1',
+                    outline: 'none',
+                    resize: 'none'
+                }}
+                />
+              <p style={{
+                    display: 'block',
+                    width: '900px',
+                    height: '120px',
+                    lineBreak: 'anywhere',
+                    fontSize: '20px',
+                    color:'white',
+                    letterSpacing: '10px',
+                    position: 'relative',
+                    top: '-20px',
+                    zIndex: '0'
```

Comment:
You may not need `position:relative` and the `z-index` here if you use `pointer-events: none`, which makes it non-selectable.